collect.py

```
@@ -0,0 +1,23 @@
+# src/collect.py
+# Scraper stub: downloads the Wikipedia page and extracts the first wikitable.
+# Requires internet; run this locally to fetch the live dataset.
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+
+WIKI_URL = "https://en.wikipedia.org/wiki/List_of_helicopter_prison_escapes"
+
+def fetch_table():
+    resp = requests.get(WIKI_URL)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)